library/core/src/ops/function.rs

```diff
@@ -59,6 +59,12 @@
         Args = "()",
         note = "wrap the `{Self}` in a closure with no arguments: `|| {{ /* code */ }}`"
     ),
+    on(
+        _Self = "unsafe fn",
+        note = "unsafe function cannot be called generically without an unsafe block",
```

Comment:
nit: s/function/functions (or unsafe function -> an unsafe function, no preference which)

also nit: do you think the word "generically" serves any purpose here? i'm actually not sure if it's better to leave it out, haha. do whatever you feel is best.